frontend/src/metabase/api/dashboard.ts

```diff
@@ -142,8 +142,15 @@ export const dashboardApi = Api.injectEndpoints({
       }),
       invalidatesTags: (_, error) =>
         invalidateTags(error, [listTag("public-dashboard")]),
+      transformResponse: ({ uuid }, _meta, { id }) => ({
+        id,
+        uuid,
+      }),
```

Comment:
I believe this is adding the `id` from the payload to the response, right? I assume our endpoint only returns the `uuid` and not the `id`?

Reply:
yep it only returns the `uuid`